plugin/pxr/maya/lib/usdMaya/testenv/testUsdExportPackage.py

```diff
@@ -31,6 +31,11 @@ def setUpClass(cls):
         standalone.initialize('usd')
         cmds.loadPlugin('pxrUsd')
 
+        # Deprecated since version 3.3: assertNotRegexpMatches has been renamed
+        # to assertNotRegex()
+        if sys.version_info.major < 3 or sys.version_info.minor < 3:
+            cls.assertNotRegex = cls.assertNotRegexpMatches
```

Comment:
The "or" doesn't seem correct to me. Instead of a python version check can you instead use the hasattr() method?